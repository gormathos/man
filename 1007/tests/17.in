62;22;14;-44;5;-32;-13;2;-46;1;45;7;5;63;-89;-2;36;-50;-87;-55
72;31;80;-71;5;-93;-99;100;17;-95;-24;7;5;-56;50;-52;75;-96;-97;-53
-43;7;-10;43;5;1;-5;29;-85;60;-30;7;5;84;30;-100;-63;57;58;-28
-55;14;-48;0;5;-9;71;80;98;-23;-100;7;5;83;-23;43;16;-72;-53;-20
-53;-88;-46;76;5;-83;95;93;37;61;24;7;5;-85;-84;92;8;-75;-72;44
-36;42;-68;-88;5;92;85;84;33;-43;-31;7;5;-55;-9;-7;64;-21;-98;-98
-31;-3;0;-17;5;49;32;-57;-51; ;54;7;5;94;57;-28;-13;52;7;72
-100;35;92;22;5;-72;-48;55;-27;-78;45;7;5;4;74;86;-52;F;54;5
61;-5;47;61;5;-9;92;-9;5;88;-49;7;5;68;-68;-70;79;-53;-90;-54
-6;-33;50;-73;5;-63;-52;-10;-35;-92;1;7;5;88;-28;-15;-95;52;59;41
-58;-7;74;1;5;56;-70;-99;-43;-47;86;7;5;80;34;-39;-40;-28;-56;8
