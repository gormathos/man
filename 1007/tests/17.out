62;22;14;-44;-32;-13;2;-46;1;45;63;-89;-2;36;-50;-87;-55
72;31;80;-71;-93;-99;100;17;-95;-24;-56;50;-52;75;-96;-97;-53
-43;7;-10;43;1;-5;29;-85;60;-30;84;30;-100;-63;57;58;-28
-55;14;-48;0;-9;71;80;98;-23;-100;83;-23;43;16;-72;-53;-20
-53;-88;-46;76;-83;95;93;37;61;24;-85;-84;92;8;-75;-72;44
-36;42;-68;-88;92;85;84;33;-43;-31;-55;-9;-7;64;-21;-98;-98
61;-5;47;61;-9;92;-9;5;88;-49;68;-68;-70;79;-53;-90;-54
-6;-33;50;-73;-63;-52;-10;-35;-92;1;88;-28;-15;-95;52;59;41
-58;-7;74;1;56;-70;-99;-43;-47;86;80;34;-39;-40;-28;-56;8
